data/gegno/gegno intro missions.txt

```diff
@@ -432,6 +433,312 @@ mission "Giaru Gegno: Revisit"
+			`	They escort you back to your ship, and along the way, several more Scin stand guard in the same armor. Once you are at your ship, the blade is moved away from your neck. When you get a better look at it, you realize that the blade is shaped like a sickle as opposed to a sword, meaning there was little chance you could have escaped being decapitated if they had chosen to use it. Boarding your ship in a hurry, you prepare to take off, and you notice the two Scin warships also begin to power up. It's clear you've overstayed your welcome.`
+				flee
+			
+			label "silence"
+				`	The Scin stares at you for an uncomfortable length of time, then sighs. They reach into their armor and pull out a small firearm, raising it at you.`
+			choice
+				`	(Reach for your firearm.)`
+				`	(Raise your arms.)`
+			action
+				set "scin: provocative"
+			`As soon as you attempt to move, the first thing you feel is a curved blade around your neck. The Scin casually marks another note on their device, then leaves the scene, heading deeper into the vessel. Slowly turning your head, your eyes are met with the cold, expressionless look of yet another Scin's face, whom you haven't seen yet. They continue to hold the blade to your throat as they activate the elevator again, lowering you back to the surface. Their armor is considerably different from the rest, characterized by darker shades, sharper edges, and several gadgets.`
+			`	They escort you back to your ship, and along the way, several more Scin stand guard, wearing the same armor. Once you are at your ship, the blade is moved away from your neck. When you get a better look at it, you realize that the blade is shaped like a sickle as opposed to a sword, meaning there was little chance you could have escaped being decapitated if they chose to use it. You board your ship in a hurry, preparing to take off. It's clear you've overstayed your welcome.`
+				flee
+
+
+mission "Scin Intro Warships"
+	invisible
+	landing
+	source "Yiia Iyr"
+	to offer
+		or
+			has "Giaru Gegno: Quarg Contact: offered"
+			has "Giaru Gegno: Revisit: offered"
+	npc
+		to spawn
+			not "scin: gestures"
+			not "scin: provocative"
+		government "Gegno Scin"
+		personality launching uninterested
+		system "Ilirco"
+		fleet
+			names "gegno scin"
+			variant
+				"Coesite" 2
+	on enter
+		fail
+
+mission "Scin Intro Warships Hostile"
+	invisible
+	landing
+	source "Yiia Iyr"
+	to offer
+		or
+			has "Giaru Gegno: Quarg Contact: offered"
+			has "Giaru Gegno: Revisit: offered"
+	npc
+		to spawn
+			not "scin: gestures"
+			has "scin: provocative"
+		government "Gegno Scin (Combative)"
+		personality launching uninterested
+		system "Ilirco"
+		fleet
+			names "gegno scin"
+			variant
+				"Coesite" 2
+	on enter
+		fail
+
+
+# Debating on removing this, as it seems to be a bit too hand-holdey, and the above is already reason for the player to explore more.
```

Comment:
I'm in favor of leaving it in, personally. I think we (as people who are quite involved with the game and have spent a lot of time playing and working on it) can pretty easily overestimate how obviously our writing can suggest player action.

For the casual player who may not know what to do, including this mission is helpful, and not including it could lead to confusion and frustration. For the experienced or more intuitive player, including this mission might not be necessary, but I doubt this would be explicitly annoying (especially since it's just a one-time reminder/indicator). This take might show my bias as a Nintendo guy (their games often have a lot of extra reminders/indicators that would be unnecessary to experienced gamers), but that's just how I'm thinking. Ultimately up to you!